tests/lib/rules/indent.js

```diff
@@ -2007,12 +2032,14 @@ ruleTester.run("indent", rule, {
             "  index;\n" +
             "});\n",
             output:
-            "[a, b, \nc].forEach((index) => {\n" +
+            "[a, b, \n" +
+            "    c].forEach((index) => {\n" +
```

Comment:
This would be a breaking change for this rule based on the previous behavior. I am not sure whats the best way to tackle this.